lib/auth/managed_authenticator.rb

```diff
@@ -113,16 +113,16 @@ def after_authenticate(auth_token, existing_account: nil)
     result
   end
 
-  def after_create_account(user, auth)
-    auth_token = auth[:extra_data]
+  def after_create_account(user, auth_result)
```

Comment:
@davidtaylorhq I've made this change to make it clear what this variable is (compare existing usage in spec for this class below).